Add russian translate

RikiDays opened the pull request.

RikiDays committed:
Add russian translate

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+
+"_comment": "STATIONS",
+
+	"block.mmorpg.teleporter": "Картографическое устройство",
```
Сложно, скучно и главное непонятно.
В том же ПоЕ это слово перевели так
```suggestion
	"block.mmorpg.teleporter": "Машина картоходца",
```

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+{
+
```
А где потерял первые 3 строчки с советами по написанию перевода?

RikiDays replied:
Измененно

P.S. Изначальный перевод был для беты, а данный текст появился в новой версии, так что большинство строк не на своём месте

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"death.attack.mmorpg": "%1$s был убит %2$s",
+	"block.mmorpg.harvest.purple": "Хаотические семена",
+	"block.mmorpg.harvest.green": "Первозданные семена",
+	"block.mmorpg.harvest.blue": "Прозрачные семена",
```
Я хоть и уже 60 часов вбухал в игру, все равно не понимаю что это за семена.
В старом переводе они переведены как "[Сияющие](https://github.com/RobertSkalko/Mine-And-Slash-Rework/blob/bf7fabf66ac8f61673a24a900281ea73ab75689a/src/main/resources/assets/mmorpg/lang/ru_ru.json#L6)", расскажешь почему именно прозрачные?

RikiDays replied:
Измененно 

P.S. Изначальный перевод был для беты, а данный текст появился в новой версии, так что большинство строк не на своём месте

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+
+"_comment": "ITEM TOOLTIPS",
+
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
```
Чем короче фраза тем проще имхо, я за старый вариант
```suggestion
	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри станции модификации снаряжения",
```

RikiDays replied:
Как на счёт "Модифицируемый в станции для снаряжения"? более коректный перевод если судить по item_modifiable_in_station

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+"_comment": "ITEM TOOLTIPS",
+
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
+	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
```
Мои 5 копеек, опять чем короче тем имхо проще
```suggestion
	"mmorpg.tooltip.lootbox": "Что же внутри?",
```

RikiDays replied:
Таким образом можно сделать ещё короче "Что внутри?" максимально коротко и ясно

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
+	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
+	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
```
Вопрос интересный, это именно полоса опыта (то что под хотбаром) или это именно уровень из мода? Если первое, то я бы оставил так
```suggestion
	"mmorpg.tooltip.player_levelup": "Если ваша полоса опыта заполнена, используйте этот предмет для повышения уровня!",
```

RikiDays replied:
Это моё виденье данного слова, всегда так говорил

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
+	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
+	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
+	"mmorpg.tooltip.capacitor": "Поместите в станцию ремонта, чтобы уменьшить потребление топлива.",
```
Судя по всему это именно обще-используемый термин в русском переводе
```suggestion
	"mmorpg.tooltip.capacitor": "Поместите в станцию починки, чтобы уменьшить потребление топлива.",
```

RikiDays replied:
Не считаю нужным. Только в случае если сама станция называется не станция ремонта, а станция починки

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
+	"mmorpg.tooltip.capacitor": "Поместите в станцию ремонта, чтобы уменьшить потребление топлива.",
+	"mmorpg.tooltip.capacitor2": "Множитель топлива",
+	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в гнездо слота",
```
Или в крайнем случае гнездо предмета
```suggestion
	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в гнездо",
```

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+
+	"item.mmorpg.jewel.dex": "Изумрудный самоцвет",
+	"item.mmorpg.jewel.str": "Метеоритный самоцвет",
+	"item.mmorpg.jewel.int": "Звездная пыль",
```
Надо придерживаться единобразия, чтобы везде было "<прилагательное> самоцвет"
```suggestion
	"item.mmorpg.jewel.int": "Звездный самоцвет",
```

Из других вариантов: Космический, Астральный, Небесный

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.jewel.dex": "Изумрудный самоцвет",
+	"item.mmorpg.jewel.str": "Метеоритный самоцвет",
+	"item.mmorpg.jewel.int": "Звездная пыль",
+	"item.mmorpg.jewel.unique_crafted": "Уникальный самоцвет Рунной мастерской",
```
```suggestion
	"item.mmorpg.jewel.unique_crafted": "Уникальный самоцвет рунического мастерства",
```

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.stone.4": "Легендарный камень",
+	"item.mmorpg.stone.5": "Мифический камень",
+	"item.mmorpg.map": "Карта приключений",
+	"item.mmorpg.stat_soul": "Душа характеристики снаряжения",
```
Я случайно задумался, а что вообще слово Stat делает в английском переводе. Вроде ни-ху-я, предлагаю убрать.
```suggestion
	"item.mmorpg.stat_soul": "Душа снаряжения",
```

RikiDays replied:
Изменено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.stone.5": "Мифический камень",
+	"item.mmorpg.map": "Карта приключений",
+	"item.mmorpg.stat_soul": "Душа характеристики снаряжения",
+	"item.mmorpg.set_map": "Карта установки",
```
Это какая-то штука для разработчиков вроде как, я не разбирался особо, а слово "утилита" особо не переводится
```suggestion
	"item.mmorpg.set_map": "Утилита настройки карт",
```

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.map": "Карта приключений",
+	"item.mmorpg.stat_soul": "Душа характеристики снаряжения",
+	"item.mmorpg.set_map": "Карта установки",
+	"item.mmorpg.loot_bag": "Сумка добычи снаряжения",
```
К единому стилю
```suggestion
	"item.mmorpg.loot_bag": "Сумка со снаряжением",
```

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.stat_soul": "Душа характеристики снаряжения",
+	"item.mmorpg.set_map": "Карта установки",
+	"item.mmorpg.loot_bag": "Сумка добычи снаряжения",
+	"item.mmorpg.map_bag": "Сумка карт",
```
```suggestion
	"item.mmorpg.map_bag": "Сумка с картами",
```

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.set_map": "Карта установки",
+	"item.mmorpg.loot_bag": "Сумка добычи снаряжения",
+	"item.mmorpg.map_bag": "Сумка карт",
+	"item.mmorpg.currency_bag": "Сумка валюты",
```
```suggestion
	"item.mmorpg.currency_bag": "Сумка с валютой",
```

RikiDays replied:
Исправлено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.map_bag": "Сумка карт",
+	"item.mmorpg.currency_bag": "Сумка валюты",
+	"item.mmorpg.newbie_gear_bag": "Сумка новичка",
+	"item.mmorpg.tutorial_book": "Учебник эпохи изгнания",
```
Хоть предмета в игре нет, я бы такие вещи называл пафосно
```suggestion
	"item.mmorpg.tutorial_book": "Учение об Эпохе Изгнания",
```

RikiDays replied:
Изменено

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.currency_bag": "Сумка валюты",
+	"item.mmorpg.newbie_gear_bag": "Сумка новичка",
+	"item.mmorpg.tutorial_book": "Учебник эпохи изгнания",
+	"item.mmorpg.map_back_portal": "Жемчуг отхода с карты",
```
Тут я сделал вывод что ты скорее всего сам не читал свой перевод, но если просто не знаешь английского, то тогда вопрос снимается (но тогда надо вдвойне ответственно подходить к этому вопросу)

Изменения тут оставляю на твою совесть
```suggestion
	"item.mmorpg.map_back_portal": "Жемчуг отхода с карты",
```

RikiDays replied:
С анлиским на самом деле не всё так хорошо, в следующих тестах решу нужны ли здесь правки

SaloEater commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"item.mmorpg.increase_rarity_nearest_entity": "Увеличение редкости ближайшего существа",
+	"item.mmorpg.mat.infused_iron": "Наполненное железо",
+	"item.mmorpg.mat.crystallized_essence": "Кристаллизованная сущность",
+	"item.mmorpg.mat.golden_orb": "Золотой сфер",
```
Ну это беспредел уже

RikiDays replied:
Исправлено

SaloEater reviewed (changes requested):
Подозрительно сырой перевод. Пройдись сначала сам и поправь хотя бы все очевидные ошибки, а потом посмотрим еще раз

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+    "mmorpg.stat.all_all_damage": "Весь урон",
+    "mmorpg.stat.all_any_wep_damage": "Урон любым оружием",
+    "mmorpg.stat.all_attributes": "Все атрибуты",
+    "mmorpg.stat.all_chaos_damage": "Хаос урон",
```
```suggestion
    "mmorpg.stat.all_chaos_damage": "Урон хаосом",
```
all_fire_damage - Огненный урон
all_chaos_damage - Урон хаосом

RikiDays replied:
Судя по твоему комменту я не прошёл тест на робота

Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+    "mmorpg.stat.all_attributes": "Все атрибуты",
+    "mmorpg.stat.all_chaos_damage": "Хаос урон",
+    "mmorpg.stat.all_dot_damage": "Урон со временем",
+    "mmorpg.stat.all_elemental_damage": "Элементальный урон",
```
Чаще вижу, что переводят как стихии.

RikiDays replied:
Имеется ввиду "Урон стихиями"?

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat.cast_speed": "Скорость каста",
+	"mmorpg.stat.cd_ticks": "Тики перезарядки",
+	"mmorpg.stat.cdr": "Снижение перезарядки",
+	"mmorpg.stat.chaining_cdr": "Снижение перезарядки цепляющих заклинаний",
+	"mmorpg.stat.chaining_spell_dmg": "Урон от цепляющих заклинаний",
```
тег chaining говорит о том, что заклинание бьет **по цепи** (последовательно)

RikiDays replied:
Исправлено

P.S. Надеюсь данный перевод удовлетворит

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat.cdr": "Снижение перезарядки",
+	"mmorpg.stat.chaining_cdr": "Снижение перезарядки цепляющих заклинаний",
+	"mmorpg.stat.chaining_spell_dmg": "Урон от цепляющих заклинаний",
+	"mmorpg.stat.chance_of_blind": "§7Ваши §5☀ Хаос§7 Атаки имеют §a[VAL1]§7% шанс на наложение Слепоты",
```
```suggestion
	"mmorpg.stat.chance_of_blind": "§7Ваши атаки §5☀ Хаосом§7 имеют §a[VAL1]§7% шанс на наложение Слепоты",
```
(это не единственный случай)

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat.chance_of_torment": "§7Ваши §d☀ Элементальные§7 Атаки имеют §a[VAL1]§7% шанс на наложение Мучения",
+	"mmorpg.stat.chaos_any_wep_damage": "Урон Хаосом от оружия",
+	"mmorpg.stat.chaos_dot_damage": "Урон Хаосом со временем",
+	"mmorpg.stat.chaos_health_leech": "§7Поглощение §a[VAL1]§7% вашего §5☀ Хаос§7 Урона как здоровья",
+	"mmorpg.stat.chaos_mana_leech": "§7Поглощение §a[VAL1]§7% вашего §5☀ Хаос§7 Урона как маны",
```
```suggestion
	"mmorpg.stat.chaos_health_leech": "§7Поглощение §a[VAL1]§7% вашего урона §5☀ Хаосом§7 как здоровья",
	"mmorpg.stat.chaos_mana_leech": "§7Поглощение §a[VAL1]§7% вашего урона §5☀ Хаосом§7 как маны",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat.out_of_combat_regen": "Восстановление вне боя",
+	"mmorpg.stat.phys_to_all": "Физический урон во все виды урона",
+	"mmorpg.stat.phys_to_chaos": "Физический урон в урон Хаоса",	
+	"mmorpg.stat.phys_to_elemental": "Физический урон в урон стихиям",
```
```suggestion
	"mmorpg.stat.phys_to_elemental": "Физический урон в урон Стихиями",
```
С большой буквы пишу лишь потому, что в остальных подобных статах тип урона с большой

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"spell.desc.arrow_totem": "Призывает тотем, который стреляет стрелами, нанося [calc:arrow_totem] §6☀ Физического§7 урона",
+	"spell.desc.astral_totem": "Призывает тотем, который восстанавливает [calc:totem_mana] маны союзникам вокруг него.",
+	"spell.desc.awaken_mana": "Восстанавливает [calc:awaken_mana] маны.",
+	"spell.desc.black_hole": "Призывает темную сферу, притягивающую ближайших врагов и наносящую [calc:black_hole]§5☀ Хаоса§7 урона при исчезновении.",
```
```suggestion
	"spell.desc.black_hole": "Призывает темную сферу, притягивающую ближайших врагов и наносящую [calc:black_hole] урона §5☀ Хаосом§7 при исчезновении.",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"spell.desc.black_hole": "Призывает темную сферу, притягивающую ближайших врагов и наносящую [calc:black_hole]§5☀ Хаоса§7 урона при исчезновении.",
+	"spell.desc.boomerang": "Наносит удар врагам снарядом, который наносит [calc:boomerang] §6☀ Физического§7 урона",
+	"spell.desc.chain_lightning": "Наносит врагам цепной удар молнией, нанося [calc:lightning_spear] §e☀ Молнии§7 урона",
+	"spell.desc.chaos_totem": "Призывает тотем, вызывающий хаотические метеоры, наносящие [calc:chaos_totem] §5☀ Хаоса§7 урона в области.",
```
```suggestion
	"spell.desc.chaos_totem": "Призывает тотем, вызывающий хаотические метеоры, наносящие [calc:chaos_totem] урона §5☀ Хаосом§7 по области.",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"spell.desc.charged_bolt": "Выстреливает замедляющую заряженную стрелу, проходящую сквозь врагов и наносящую [calc:charged_bolt] §e☀ Молнии§7 урона в радиусе по ее пути.",
+	"spell.desc.chilling_field": "Создает облако морозного холода, повреждая врагов на [calc:chilling_field]§b☀ Холода§7 каждую секунду и накладывая замедление.",
+	"spell.desc.chilling_touch": "Наносит удар по врагам спереди на [calc:chilling_touch] §b☀ Холода§7, приказывая призванным атаковать их.",
+	"spell.desc.curse_of_agony": "Проклинает врагов Проклятием Агонии и наносит [calc:curse] §5☀ Хаоса§7 урона",
+	"spell.desc.curse_of_despair": "Проклинает врагов Проклятием Отчаяния и наносит [calc:curse] §5☀ Хаоса§7 урона",
+	"spell.desc.curse_of_weak": "Проклинает врагов Проклятием Слабости и наносит [calc:curse] §5☀ Хаоса§7 урона",
```
```suggestion
	"spell.desc.curse_of_agony": "Проклинает врагов Проклятием Агонии и наносит [calc:curse] урона §5☀ Хаосом§7",
	"spell.desc.curse_of_despair": "Проклинает врагов Проклятием Отчаяния и наносит [calc:curse] урона §5☀ Хаосом§7",
	"spell.desc.curse_of_weak": "Проклинает врагов Проклятием Слабости и наносит [calc:curse] урона §5☀ Хаосом§7",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"spell.desc.meteor_arrow": "Выстреливает стрелу, которая наносит [calc:meteor] §6☀ Физического§7 урона и призывает метеор, наносящий [calc:meteor] §c☀ Огненного§7 урона",
+	"spell.desc.nature_balm": "Дает бафф, который лечит ближайших союзников на [calc:nature_balm] каждую секунду.",
+	"spell.desc.pet_basic": "Базовая атака, наносящая [calc:pet_basic] §6☀ Физического§7 урона одному врагу.",
+	"spell.desc.poison_ball": "Выбрасывает шар яда, нанося [calc:poisonball] §5☀ Хаоса§7 урона",
+	"spell.desc.poison_cloud": "Выбрасывает ядовитый газ, нанося [calc:poison_cloud] §5☀ Хаоса§7 урона ближайшим врагам.",
+	"spell.desc.poison_trap": "Бросает ловушку, которая активируется при приближении врага, нанося [calc:ranger_trap]§5☀ Хаоса§7 урона в области вокруг себя.",
```
```suggestion
	"spell.desc.poison_ball": "Выбрасывает шар яда, нанося [calc:poisonball] урона §5☀ Хаосом§7",
	"spell.desc.poison_cloud": "Выбрасывает ядовитый газ, нанося [calc:poison_cloud] урона §5☀ Хаосом§7 ближайшим врагам.",
	"spell.desc.poison_trap": "Бросает ловушку, которая активируется при приближении врага, нанося [calc:ranger_trap] урона §5☀ Хаосом§7 в области вокруг себя.",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"spell.desc.song_of_perseverance": "Дает стопку Упорства всем союзникам вокруг вас.",
+	"spell.desc.song_of_valor": "Дает стопку Мужества всем союзникам вокруг вас.",
+	"spell.desc.song_of_vigor": "Дает стопку Бодрости всем союзникам вокруг вас.",
+	"spell.desc.spider_basic": "Базовая атака, наносящая [calc:spider_pet_basic] §5☀ Хаоса§7 урона одному врагу.",
```
```suggestion
	"spell.desc.spider_basic": "Базовая атака, наносящая [calc:spider_pet_basic] урона §5☀ Хаосом§7 одному врагу.",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"spell.desc.summon_zombie": "Призывает Зомби, чтобы помочь вам в бою.",
+	"spell.desc.taunt": "Кричит, заставляя ближайших врагов атаковать вас. Генерирует [calc:taunt] угрозы.",
+	"spell.desc.teleport": "Телепортирует вас в направлении, куда вы смотрите.",
+	"spell.desc.thorn_bush": "Призывает колючий куст, наносящий [calc:thorn_bush]§5☀ Хаоса§7 урона в области каждую секунду.",
```
```suggestion
	"spell.desc.thorn_bush": "Призывает колючий куст, наносящий [calc:thorn_bush] урона §5☀ Хаосом§7 в области каждую секунду.",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat_desc.alchemy_triple_drop": "Шанс профессии утроить дроп, если это произойдет.",
+	"mmorpg.stat_desc.all_all_damage": "Увеличивает всю наносимую урон определенного элемента, как заклинаниями, так и атаками.",
+	"mmorpg.stat_desc.all_attributes": "Добавляет к СИЛ, ЛОВ, ИНТ.",
+	"mmorpg.stat_desc.all_chaos_damage": "Увеличивает всю наносимую урон хаосом, как заклинаниями, так и атаками.",
+	"mmorpg.stat_desc.all_dot_damage": "Урон от огня с течением времени увеличивает урон от ожога, например.",
+	"mmorpg.stat_desc.all_elemental_damage": "Увеличивает всю наносимую урон определенного элемента, как заклинаниями, так и атаками.",
+	"mmorpg.stat_desc.all_fire_damage": "Увеличивает всю наносимую урон огнем, как заклинаниями, так и атаками.",
+	"mmorpg.stat_desc.all_lightning_damage": "Увеличивает всю наносимую урон молнией, как заклинаниями, так и атаками.",
+	"mmorpg.stat_desc.all_physical_damage": "Увеличивает всю наносимую физический урон, как заклинаниями, так и атаками.",
+	"mmorpg.stat_desc.all_water_damage": "Увеличивает всю наносимую урон водой, как заклинаниями, так и атаками.",
```
это как

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat_desc.dodge": "Шанс игнорировать урон от атаки.",
+	"mmorpg.stat_desc.dot_dmg": "Увеличивает урон от эффектов, наносящих урон с течением времени, например, ожог.",
+	"mmorpg.stat_desc.dot_lifesteal": "Восстанавливает % урона от эффектов, наносящих урон с течением времени, в виде здоровья.",
+	"mmorpg.stat_desc.ele_atk_dmg": "Добавляет x элементарного урона при ударе оружием.",
+	"mmorpg.stat_desc.ele_axe_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_bow_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_crossbow_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_pene": "Игнорирует столько сопротивления, это работает как вычитание.",
+	"mmorpg.stat_desc.ele_resist": "Поглощает X процентов урона этого элемента.",
+	"mmorpg.stat_desc.ele_staff_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_sword_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_trident_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
```
```suggestion
	"mmorpg.stat_desc.ele_atk_dmg": "Добавляет x урона от стихий при ударе оружием.",
	"mmorpg.stat_desc.ele_axe_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_bow_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_crossbow_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_pene": "Игнорирует столько сопротивления, это работает как вычитание.",
	"mmorpg.stat_desc.ele_resist": "Поглощает X процентов урона этого элемента.",
	"mmorpg.stat_desc.ele_staff_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_sword_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_trident_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
```
урон стихиями, урон стихий, стихийный урон

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.chest_type.currency": "Валюта",
+	"mmorpg.chest_type.gear": "Экипировка",
+	"mmorpg.chest_type.gem": "Самоцвет",
+	"mmorpg.chest_type.harvest_blue_chest": "Сбор Голубого Сундука",
+	"mmorpg.chest_type.harvest_green_chest": "Сбор Зеленого Сундука",
+	"mmorpg.chest_type.harvest_purple_chest": "Сбор Фиолетового Сундука",
```
```suggestion
	"mmorpg.chest_type.harvest_blue_chest": "Голубой сундук Жатвы",
	"mmorpg.chest_type.harvest_green_chest": "Зелёный сундук Жатвы",
	"mmorpg.chest_type.harvest_purple_chest": "Фиолетовый сундук Жатвы",
```

RikiDays replied:
Исправлено

breadusofc commented on src/main/resources/assets/mmorpg/lang/ru_ru.json:
```diff
@@ -0,0 +1,2718 @@
+	"mmorpg.stat.energy_per_sec": "Энергия за секунду",
+	"mmorpg.stat.energy_regen": "Восстановление энергии",
+	"mmorpg.stat.extra_mob_drops": "Дополнительные дропы от монстров",
+	"mmorpg.stat.farming_core_drop_bonus": "Бонус ядра при фермерстве",
+	"mmorpg.stat.farming_double_drop": "Шанс на удвоенный дроп при фермерстве",
+	"mmorpg.stat.farming_exp_bonus": "Бонус опыта при фермерстве",
+	"mmorpg.stat.farming_greater_drop_bonus": "Эпический дроп при фермерстве",	
+	"mmorpg.stat.farming_lesser_drop_bonus": "Обычные выпадения",
+	"mmorpg.stat.farming_medium_drop_bonus": "Редкие выпадения",
+	"mmorpg.stat.farming_misc_drop_bonus": "Различные выпадения",
+	"mmorpg.stat.farming_triple_drop": "Шанс на тройной выпад",
```
дропы превратились в выпадения

RikiDays replied:
Исправлено

breadusofc reviewed (changes requested):
Это 100% переводил не человек.
Перепроверь профессии и заклинания.

RikiDays committed:
Test

RikiDays committed:
Patch 1.0.1

- Исправлены критические ошибки из коментариев

- Исправлены ошибки в строках:
    33, 41-42, 50-54, 56, 62, 875, 878, 925-929, 932-933, 935, 946, 990-994, 1013-1016, 1019, 1022-1023, 1047-1048, 1055-1060, 1097-1100, 1199-1200, 1202, 1254-1256, 1261-1262, 1270-1281, 1289-1290, 1308-1309, 1314-1318, 1545, 1547-1548, 1550-1551, 1553-1555, 1560-1570, 1576-1579, 1581-1582, 1586-1588, 1604, 1614-1615, 1637-1643, 1663, 1705-1708, 1710-1713, 1721, 1837, 1839, 1842-1843, 1845, 1847, 1866, 1879-1881, 2316-2327

RikiDays commented:
P.S. Спасибо breadusofc и SaloEater за найдены ошибки, некоторые из них ещё обработаю но в целом большинство исправил

RobertSkalko commented:
This is the good way to do translation. PR, let others comment and fix stuff.

I'll just wait a day or two before the last comment and merge the PR to make sure everyone had time to comment

RobertSkalko commented:
Alright, any more edits will need a new PR
